.gitignore

```diff
@@ -64,7 +64,7 @@ instance/
 .scrapy
 
 # Sphinx documentation
-docs/_build/
+docs/en/_build/
```

Comment:
We should also add docs/en/zh_cn/_build/

Reply:
docs/zh_cn/_build/

Reply:
Thanks for reminder, `docs/zh_cn/_build/` has been added.